libs/util/code-gen-types/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@amplication/code-gen-types",
-  "version": "2.0.35",
+  "version": "2.0.36-beta.0",
```

Comment:
is that correct?

Reply:
@yuval-hazaz bump version to : 2.0.36 (I will publish it after your final approve).